Update User Guide - Add support for style with properties and styling classification (#9581)
geosolutions-it/MapStore2: docs/user-guide/layer-settings.md

<img src="../img/layer-settings/classify_icon_ex.jpg" class="ms-docimage">

#### Style with properties

> [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to choose a property of a layer feature as the value for a symbolizer option. The *Style with properties* is available for *Marker*, *Icon*, *Line*, *Fill*, *Text* and *3D model* by clicking on the <img src="../img/button/options_button.jpg" class="ms-docbutton"/> button, of each symbolizer option, and choosing **Property value** option from the dropdown menu.

Reviewer: ```suggestion
[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to choose a property of a layer's feature as the value for a symbolizer option. The *Style with properties* function is available for *Marker*, *Icon*, *Line*, *Fill*, *Text* and *3D model* by clicking on the <img src="../img/button/options_button.jpg" class="ms-docbutton"/> button of each symbolizer optionsupported by this function and choosing **Property value** option from the dropdown menu.
```

<img src="../img/layer-settings/properties_style.jpg" class="ms-docimage"  style="max-width:500px;">

> By default, the **Constant value** is selected. If the user chooses the **Property value**, a dropdown menu appears and the user can select a layer feature. For example, in the fallows video, a **Property value** is selected for the `Radius` option and the *Integer* feature is added to customize the option.

Reviewer: ```suggestion
By default, **Constant value** is selected. If the user chooses **Property value**, a drop-down menu appears so that the desired layer attribute can be selected to use the necessary values from the styling property.
```

<video class="ms-docimage" controls><source src="../img/layer-settings/property-value-example.mp4"></video>

!!!warning

>     As a *Property value* only features that support `string` or `number` are available, `boolean` and `array` are not currently supported.

Reviewer: ```suggestion
    As a *Property value* only attributes of type `string` or `number` are currently supported.
```
